lib/tokenize.cpp

```diff
@@ -3407,30 +3407,21 @@ bool Tokenizer::simplifyTokens1(const std::string &configuration)
             return false;
     }
 
-    if (mTimerResults) {
-        Timer t("Tokenizer::simplifyTokens1::createAst", mSettings.showtime, mTimerResults);
-        list.createAst();
-        list.validateAst(mSettings.debugnormal);
-    } else {
+    const SHOWTIME_MODES showTime = mTimerResults ? mSettings.showtime : SHOWTIME_MODES::SHOWTIME_NONE;
```

Comment:
Pulling this out does not seem to make any sense. I think this might have been left over from when the settings were still a pointer to remove duplicated dereferencing code.